Ginger/Ginger/ApplicationModelsLib/POMModels/POMWizardLib/POMObjectsMappingWizardPage.xaml.cs

```diff
@@ -179,7 +180,27 @@ private void ElementsListCollectionChanged(object sender, NotifyCollectionChange
                 Reporter.ToLog(eLogLevel.ERROR, "POM: Learned Element Info from type was failed to be added to Page Elements", ex);
             }
         }
-        
+
+        private void UpdateElementInfoName(ElementInfo curElement)
+        {
+            try
+            {
```

Comment:
what about duplicate name handling?

Reply:
already implemented but it is handled while export to datasource

Reply:
why only on Export to Data Source? let's do it also here so on learning we will have unique names.
we still need to do all of this name fixing also on export because use may change the learned name manually and also in order to support old learned POM's

Reply:
implemented at both places